tools/internal_ci/linux/grpc_bazel_rbe_incompatible_changes.sh

```diff
@@ -15,9 +15,14 @@
 
 set -ex
 
-# TODO(jtattermusch): use the latest version of bazel
+# Use bazelisk to download the right bazel version
+wget https://github.com/bazelbuild/bazelisk/releases/download/v0.0.7/bazelisk-linux-amd64
```

Comment:
Any reason not to pull [the latest?](https://help.github.com/en/articles/linking-to-releases#linking-to-the-latest-release)

Reply:
I didn't know about that option, but on the other hand, the current version does what we need from it and pinning a version usually means fewest surprises.